lib/Doctrine/ORM/Tools/Pagination/WhereInWalker.php

```diff
@@ -28,7 +28,7 @@
 use function reset;
 
 /**
- * Replaces the whereClause of the AST with a WHERE id IN (:foo_1, :foo_2) equivalent.
+ * Appends to the whereClause of the AST a Condition "id IN (:foo_1, :foo_2)".
```

Comment:
This sentence looks odd. 

```suggestion
 * Appends a condition "id IN (:foo_1, :foo_2)" to the whereClause of the AST.
```